client/lib/analytics/index.js

```diff
@@ -160,6 +160,10 @@ function buildQuerystringNoPrefix( group, name ) {
 // this helps avoid some nasty coupling, but it's not the cleanest code - sorry.
 let mostRecentUrlPath = null;
 
+// record the last path of a `calypso_page_view` tracks event. Useful for building
+// flows of page interactions (sankey diagrams).
+let lastPageViewPath = null;
```

Comment:
It seems you can use`mostRecentUrlPath` instead of introducing a new variable, no?

Reply:
Good call! Searching for `mostRecentUrlPath` and it seems to only be referenced in client/lib/analytics/index.js. Thank you for the review @psealock!

Reply:
Removed `lastPageViewPath` and moved the assignment of `mostRecentUrlPath = urlPath;` below the `recordPageView()`.